src/Client.js

```diff
@@ -21,10 +21,32 @@ module.exports = class Client {
   }
 
   async canSendMessage() {
+    if (!this.config) {
+      throw new Error(`No config provided for client.`);
+    }
+
+    if (!this.config.baseURL) {
+      throw new Error(`The config does not contain a 'baseURL' field.`);
+    }
+
+    if (!this.config.clientId) {
+      throw new Error(`The config does not contain a 'clientId' field.`);
+    }
+
+    if (!this.config.aud) {
+      throw new Error(`The config does not contain an 'aud' field.`);
+    }
+
+    if (!(this.config.jwk || (this.config.pkcs12 && this.config.pkcs12Pass))) {
+      throw new Error(
+          `The config does not contain a 'jwk' field or a 'pkcs12' and 'pkcs12Pass' field.`,
+      );
```

Comment:
Okay one more small thought - Could we modify these error messages slightly to make the localization easier? Specifically referencing "FHIRMessagingClient"; like "FHIRMessagingClient config does not contain..." and "No config provided for FhirMessagingClient". That'd be my preference, but I'm open to alternative views!